composer.json

```diff
@@ -23,9 +23,7 @@
   "require"    : {
     "php": ">=5.4.0",
     "consolidation/robo": "~1",
-    "joomla/github": "~1.3",
-    "greencape/coding-standards": "~1",
```

Comment:
something specific? asking because I'm updating it in the `container-test` branch without any problem so far